test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
Delete unused test files. Next time you can start a new Rails app without the default testing framework by using the `-T` flag `$rails new app -T`

[OPTIONAL]